lib/content-services/src/lib/content-node-share/content-node-share.dialog.html

```diff
@@ -1,89 +1,122 @@
-                    [target]="sharedLinkInput"
-                    tabindex="0">
-                    content_copy
-                </mat-icon>
-            </mat-form-field>
-
-            <div class="adf-share-link--row">
-                <div class="adf-share-link__label">{{ 'SHARE.EXPIRES' | translate }}</div>
-                <mat-slide-toggle
-                    #slideToggleExpirationDate
-                    [disabled]="!canUpdate"
+        <mat-dialog-content>
+            <hr class="adf-share-link__seperation-line" />
+
+            <form [formGroup]="form" class="adf-share-link__form">
+
+                <div class="adf-share-link--row">
+                    <mat-icon class="adf-share-link__icon">timer</mat-icon>
+
+                    <div class="adf-share-link__label adf-sharable-link">{{ 'SHARE.LINK-EXPIRY-DATE' | translate }}</div>
+
+                    <mat-slide-toggle
+                        #slideToggleExpirationDate
+                        [disabled]="!canUpdate"
+                        color="primary"
+                        data-automation-id="adf-expire-toggle"
+                        aria-label="{{ 'SHARE.EXPIRES' | translate }}"
+                        [checked]="time.value"
+                        (change)="onToggleExpirationDate($event)">
+                    </mat-slide-toggle>
+                </div>
+
+                <div *ngIf="isExpiryDateToggleChecked" data-automation-id="adf-slide-toggle-checked"
+                class="adf-share-link__date-time-container">
+                    <mat-form-field class="adf-full-width adf-float-label" floatLabel='always'>
+                        <mat-label>{{ 'SHARE.EXPIRATION-PLACEHOLDER' | translate }}</mat-label>
+                        <mat-datetimepicker-toggle
+                            #matDatetimepickerToggle="matDatetimepickerToggle"
+                            [disabled]="time.disabled"
+                            [for]="datetimePicker"
+                            matSuffix
+                            class="adf-share-link__icon adf-share-link__calender-icon">
+                        </mat-datetimepicker-toggle>
+                        <mat-datetimepicker
+                            #datetimePicker
+                            (closed)="onDatetimepickerClosed()"
+                            [type]="type"
+                            [timeInterval]="1">
+                        </mat-datetimepicker>
+                        <input class="adf-share-link__input"
+                            #dateTimePickerInput
+                            matInput
+                            placeholder="{{ 'SHARE.EXPIRATION-PLACEHOLDER' | translate }}"
+                            [attr.aria-label]="'SHARE.EXPIRATION-LABEL' | translate"
+                            [min]="minDate"
+                            formControlName="time"
+                            [matDatetimepicker]="datetimePicker" />
```

Comment:
My IDE shows me here following error. Can you check? 
<img width="1289" alt="image" src="https://user-images.githubusercontent.com/115619721/236834852-4915e183-8b68-4592-aa39-88d18b2241ed.png">

Reply:
hi Alek, I am not able to find the error in VS code.

Reply:
I can't reproduce it in intellij anymore as well so looks like this has been resolved in meantime